src/main/java/bio/terra/common/stairway/TracingHook.java

```diff
@@ -174,6 +174,7 @@ public HookAction end(FlightContext flightContext) {
    * Gets or creates the context of the submission Span. Each Flight span is linked to the
    * submission span to correlate all Flight Spans.
    */
+  @SuppressWarnings("ReturnCount")
```

Comment:
IMHO, early return is a good pattern, not a bad one. go/tott/455

Reply:
I'd adjust the max value here rather than suppress. More than about 4 is suspicious I'd think. Default is 2.

Reply:
Similar story here; I've opted to exclude this rule.